sound/hybrid_font.h

```diff
@@ -120,6 +121,36 @@ class HybridFont : public SaberBase {
       PlayPolyphonic(effect);
     }
   }
+  
+  void CommonSwing(Effect* monophonic, Effect* polyphonic) override {
+    if (polyphonic->files_found()) {
+      if (!IsSwingPlaying()) {
+        swing_player_ = PlayPolyphonic(polyphonic);
+      }
+    } else {
+      PlayMonophonic(monophonic, &hum);
+    }
+  }
+  
+  void SetSwingVolume(float volume) override {
```

Comment:
I wonder if it would be better if this function just took the current speed, that way it could (optionally) be used by HybridFont::SB_Motion() as well.

Or maybe a speed and a multiplier?

Reply:
I'll play with that.